src/analytics.js

```diff
@@ -22,6 +22,20 @@ class Analytics {
     this._track(data)
   }
 
+  trackListSpaces (address) {
+    let data = {}
+    data.event = 'list_spaces'
+    data.properties = { address: address }
+    this._track(data)
+  }
+
+  trackGetSpace (address, name, spaceExisted) {
+    let data = {}
+    data.event = 'get_space'
+    data.properties = { address: address, name: name, profile_existed: spaceExisted }
```

Comment:
fwiw, you can do `data.properties = { address, name, profile_existed: spaceExisted }`

Reply:
Oh, I know. Not sure why it wasn't done here 🙂